Gemfile

```diff
@@ -1,8 +1,24 @@
 source 'https://rubygems.org'
 
+ruby '2.7.3'
+
+gem 'pg'
+gem 'sinatra'
+gem 'sinatra-contrib'
+gem 'sinatra-reloader'
+gem 'activerecord', '5.2.6'
+gem 'sinatra-activerecord'
+gem 'rake'
+
 group :test do
+  gem 'capybara'
   gem 'rspec'
   gem 'rubocop', '0.79.0'
   gem 'simplecov', require: false
   gem 'simplecov-console', require: false
 end
+
+group :development do
+  gem 'tux'
```

Comment:
I've just had a quick look at what tux is for. If you were finding it useful, would love to see how you're using it.